package.json

```diff
@@ -20,11 +20,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "handlebars": "^4.1.0",
-    "marked": "^0.6.1",
+    "coveralls": "^3.0.3",
+    "handlebars": "^4.1.2",
+    "marked": "^0.6.2",
     "minimist": "^1.2.0",
     "moment": "^2.24.0",
-    "source-map-support": "^0.5.10"
+    "nyc": "^13.3.0",
+    "source-map-support": "^0.5.10",
+    "tap": "^12.1.0"
```

Comment:
Tiny comment here - let's put dev deps into `devDependencies`, such as `coveralls`, `nyc` and `tap`. Do we really need them all?

Reply:
Moved coveralls, nyc and tap to devDeps

Reply:
Thanks!
I still wonder - we didn't use to have `nyc` and `coveralls` prior to this PR, and I don't see them used by any `script`. Do we need them then?

Reply:
Seems like there might be some issue with Wizard, Kosta is looking into it.

Reply:
Downgraded the cli to version 1.163.1 and then ran wizard, seems to have done the trick.